test/modules/core/core/animation-loop.spec.js

```diff
@@ -71,3 +71,78 @@ test('WebGL#AnimationLoop redraw', t => {
     }
   }).start();
 });
+
+test('WebGL#AnimationLoop two start()s should only run one loop', t => {
+  if (typeof document === 'undefined') {
+    t.comment('browser-only test');
+    t.end();
+    return;
+  }
+
+  const {gl} = fixture;
+  let initializeCalled = 0;
+
+  const animationLoop = new AnimationLoop({
+    gl,
+    onInitialize: () => {
+      initializeCalled++;
+    },
+    onRender: () => {
+      animationLoop.stop();
+      // FIXME: this is not directly test that only one loop has been started
+      t.is(initializeCalled, 1, 'onInitialize called');
```

Comment:
Note the `FIXME` here. Any ideas on how we could directly test that only one loop is running?